xml/cl.xml

```diff
@@ -1004,6 +1004,12 @@ server's OpenCL/api-docs repository.
         <enum bitpos="0"            name="CL_MEM_ALLOC_WRITE_COMBINED_INTEL"/>
     </enums>
 
+    <enums start="0x10000" end="0x1FFFF" name="cl_khronos_vendor_id" vendor="Khronos">
```

Comment:
I've just realized that `cl_khronos_vendor_id` isn't a defined in the spec (or this XML file), because a `cl_uint` is returned for this query.

Could put `cl_uint` here and that doesn't affect the current spec, but all the other enums have a type.  Adding a type in a maintenance spec release seems a bit sneaky though.

Reply:
To avoid any risk I made `cl_khronos_vendor_id` the comment attribute and `cl_uint` the name attribute. Based on the [schema](https://github.com/KhronosGroup/OpenCL-Docs/blob/master/xml/registry.rnc#L176) for `<enums>` that seems valid, didn't update the spec language but think it still makes sense.

Reply:
Related PR https://github.com/KhronosGroup/OpenCL-Docs/pull/235

Reply:
updated to use expeimental enum from [235](https://github.com/KhronosGroup/OpenCL-Docs/pull/235) and headers [MR 75](https://github.com/KhronosGroup/OpenCL-Headers/pull/75)